app/posts/manage-teacher-training-applications/2020-12-18-search-by-candidate-name-or-reference.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Search by candidate name or reference
+description: Let users search the applications list by candidate name or reference
+date: 2020-12-18
+---
+
+We introduced a search to allow users to search the applications list by candidate name or number.
```

Comment:
We have had the ability to search by name for a long time.

This work is more adding in the ability to search by reference number and improving the interaction while we’re at it.

Reply:
Okay, I've updated to this:

>We updated the search to allow users to find applications by candidate number in addition to candidate name.